andino_description/test/test_xacro_processing.py

```diff
@@ -0,0 +1,13 @@
+import os
+import pytest
+import xacro
+
+def test_xacro_processing():
+    # Get the file path
+    xacro_file_path = os.path.join(os.path.dirname(__file__), '..', 'urdf', 'andino.urdf.xacro')
```

Comment:
is it possible to use here ament_index to get the package path and later on the xacro file path?


```python
from ament_index_python.packages import get_package_share_directory

get_package_share_directory("package_a")
```

Reply:
Actually it is possible, I didn't think of it but it is the preferred approach, thanks Franco!

Reply:
Thanks for addressing this. This way we can directly check xacro on install folder. So the installation process is also tested for free.

Reply:
Happy to help!